index.js

```diff
@@ -35,7 +37,14 @@ class HomeBridgeTasmotaAirconHTTP {
       this.switchTurboService = this._setupSwitchTurboService(api.hap);
       this.switchEconoService = this._setupSwitchEconoService(api.hap);
       this.switchQuietService = this._setupSwitchQuietService(api.hap);
+
+      //Get Previous state from storage
+      storage.initSync({'dir': path.join(api.user.storagePath(), 'HomeBridgeTasmotaAirconHTTP')});
+      this.state = storage.getItemSync(this.name) || this._initialState(config);
     }
+
+    //Get Temperature from tasmota task
```

Comment:
This goes for everywhere: Please add a space after "//". Also in this case it's obvious what's going to happen, since the method names has very good name, except that the method name should probably...

1. be prefixed with "_", since it's not something that should be called from "outside" of this plugin.
2. Have a name that indicates that it will do the same thing over and over again.

So, since I really like the name, I would rather do something like this:

```
// Get Temperature from Tasmota every two minutes
setInterval(() => this.getTemperatureFromTasmota(), 2 * 60 * 1000);
```

This also makes it much easier to test getTemperatureFromTasmota().